OpenRA.Mods.Cnc/Activities/Leap.cs

```diff
@@ -42,8 +44,8 @@ public Leap(Actor self, Actor target, Armament a, WDist speed, WAngle angle, Bit
 			this.angle = angle;
 			this.damageTypes = damageTypes;
 			mobile = self.Trait<Mobile>();
+			notifyMoving = self.TraitsImplementing<INotifyMoving>().ToArray();
 			mobile.SetLocation(mobile.FromCell, mobile.FromSubCell, targetMobile.FromCell, targetMobile.FromSubCell);
```

Comment:
Can you please try moving this stuff too?

Reply:
Just this line or the stuff below it as well?

Reply:
The stuff below too, as that depends on or changes the state of the actor at the time the activity first runs.

Reply:
Updated, but I kept the changes in a separate commit as the number of changes this required made me a bit uncomfortable about squashing it right away.